src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,589 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses IntAct data and maps the interaction types to BEL.
+
+Run with ``python -m bio2bel.sources.intact``
+"""
```

Comment:
Information here, same drill as BioGRID. What's in the database, how is it different from BioGRID, how do you go about mapping to BEL, when are there overlaps that you can re-use code you wrote for BioGRID? You can reference the fact that you did BioGRID first, but what were the specific problems with the relationships in IntAct?

Comment on the fact that IntAct also has negative data, which could be mapped to negative BEL (see/reference https://github.com/pybel/pybel/pull/408) - before this is done I promise I will submit it as a proper BEP to https://bep.bel.bio.

Reply:
Comment on what types of entities you were able to deal with (proteins, some complexes that could be mapped to reasonable identifiers) and what were concerns (how did you deal with isoforms) and what entity types/identifiers did you throw away because they did not normalize well into the uniprot world?

Reply:
Give summary of BEL - nodes, edges, citations from `BELGraph.summarize()`. Maybe also give summary of PTMs (there's a pybel function for that too, somewhere in `pybel.struct`)

Reply:
I've written the docstrings: 8649def7765ef29f557c0660762292a058a3ed24

For the same reasons as yesterday, I cannot run the scripts. Could you try again and see if you can run it? Therefore, I have also not been able to includ the summaries yet.

Reply:
Did the fixes I suggested enable you to run the script?

Reply:
I can run the script, but the output is again
```python-traceback
intact v2020-03-31
Number of Nodes: 0
Number of Edges: 0
Number of Citations: 0
Number of Authors: 0
Network Density: 0.00E+00
Number of Components: 0
Number of Warnings: 0
```